cmd/query/app/flags.go

```diff
@@ -40,6 +40,9 @@ const (
 	queryHostPort           = "query.host-port"
 	queryPort               = "query.port"
 	queryPortWarning        = "(deprecated, will be removed after 2020-08-31 or in release v1.20.0, whichever is later)"
+	queryHOSTPortWarning    = "(deprecated, will be removed after 2020-11-30 or in release v1.20.0, whichever is later)"
```

Comment:
Actually, this is now wrong again. It should be at least one version after the next one, so that people get enough time to migrate/remove. 

The date is a bit arbitrary: we aim to have one release per month, so, a couple of months in the future means two versions, which matches the "next+1" idea.

Reply:
The date is not arbitrary either, we agreed on 3 months or v+2, whichever is later, see #2113 (which needs to be moved to CONTRIBUTING)